lib/styles.js

```diff
@@ -347,4 +347,118 @@ module.exports = function (expect) {
             }
         }, this);
     });
+
+    expect.addStyle('arrow', function (options) {
+        options = options || {};
+        var styles = options.styles || [];
+        var i;
+        this.nl(options.top || 0).sp(options.left || 0).text('┌', styles);
+        for (i = 1 ; i < options.width ; i += 1) {
+            this.text(i === options.width - 1 && options.direction === 'up' ? '▷' : '─', styles);
+        }
+        this.nl();
+        for (i = 1 ; i < options.height - 1 ; i += 1) {
+            this.sp(options.left || 0).text('│', styles).nl();
+        }
+        this.sp(options.left || 0).text('└', styles);
+        for (i = 1 ; i < options.width ; i += 1) {
+            this.text(i === options.width - 1 && options.direction === 'down' ? '▷' : '─', styles);
+        }
+    });
```

Comment:
If we can make it a bit more generic, and if unexpected-htmllike/-react cannot reach this feature through the array-like type, I agree.